### ISSUE TITLE
Python error: invalid_grant, error_description: Token has been expired or revoked.

### ISSUE THREAD

**Issue by Heryx** [EnMAP-Box/enmap-box#296]
Hello everyone,

since I upgraded QGIS from 2.24 to 2.28 I am getting this error:

`('invalid_grant: Token has been expired or revoked.', {'error': 'invalid_grant', 'error_description': 'Token has been expired or revoked.'})`

I own QGIS Version 3.28.1-Florence and Python Version 3.9.5.

Thank you very much.

**Comment by janzandr** [comment]
When is that error showing up? Directly after starting the EnMAP-Box?
It seams to be related to your Google API authentification, which is only needed, if you want to use the GEE Time Series Explorer app.
Here I found something on stackoverflow: https://stackoverflow.com/questions/66058279/token-has-been-expired-or-revoked-google-oauth2-refresh-token-gets-expired-i

**Comment by Heryx** [comment]
The error pops up when I try to upload satellite images (Sentinel, Landsat etc). I have tried to re-authenticate the account both in the QGIS python console and from Osgeo via the commands:

`import ee`
`ee.Authenticate()`

but it returns this error:

```
File "C:\OSGeo4W\apps\Python39\lib\site-packages\ee\oauth.py", line 294, in _load_app_default_credentials
    raise Exception('gcloud command not found. ' + tip) from e
Exception: gcloud command not found. Please ensure that gcloud is installed.
More information: https://developers.google.com/earth-engine/guides/python_install
```

As you suggested, I looked at the directions on stackoverflow. Apparently the token is valid for 7 days, unfortunately it doesn't say how to fix it for QGIS.

![image](https://user-images.githubusercontent.com/11267377/207396909-549928a9-c9fe-4d88-825a-22571554e3ba.png)
